src/data-visualisation/accessibility/design-a11y.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: article
+title: "Designing for accessibility"
+description: "How do we design for accessibiliity at the NHSBSA?"
+status: DRAFT
+tags: [data-viz, data-viz-a11y]
+order:
+    data-viz: 2
+    data-viz-a11y: 3
+---
+## Designing for accessibility  
+
+The UK Home Office created ['Designing for accessibility' posters.][home office posters] 
+
+The posters include Dos and Don’ts for individuals:  
+- on the autistic spectrum
+- who use screen readers
+- with low vision
+- with dyslexia
+- with physical or motor disabilities
+- who are d/Deaf or hard of hearing
+- with anxiety  
+  
+## Content design standards on numbers  
+
+The following tips are the most relevant to data visualisation.  
+
+Numbers:  
+- Use ‘one’ unless you’re talking about a step, a point in a list or another situation where using the numeral makes more sense 
+- Write all other numbers in numerals, including 2 to 9, except where it’s part of a common expression like ‘one or two of them’ 
+- Insert a comma to add clarity for values over 1,000, for example, 9,000
+- Use spacing between area and exchange codes for telephone numbers, for example, 0191 123 4567  
+
+Money:
+- Use the ‘£’ symbol  
+- Do not use decimals unless pence are included, for example, '£75.50' but not '£75.00' 
+- Separate large amounts of money using commas, for example, '£10,000' not '£10000' 
+- For amounts in millions or billions, format as '£X million' or '£X billion', not written in full 
+- Do not use '£0.XX millions' for amounts less than £1 million  
+
+::: details Source
+ 
+[Content Design Team ][numbers 1]
+
+::: 
+
+
```

Comment:
Not a biggy but - markdown format usually follows a single consecutive blank line between text/elements.